core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockCromwellSwaggerClient.scala

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import cromwell.client.model.ValueType.TypeNameEnum
+import cromwell.client.model.{ToolInputParameter, ToolOutputParameter, ValueType, WorkflowDescription}
+import org.broadinstitute.dsde.rawls.model.UserInfo
+
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable
+import scala.collection.JavaConverters._
+import scala.util.Try
+
+class MockCromwellSwaggerClient extends CromwellSwaggerClient("fake/path") {
+
+  val workflowDescriptions: mutable.Map[String, WorkflowDescription] =  new TrieMap()
+
+  override def describe(userInfo: UserInfo, wdl: String): Try[WorkflowDescription] = {
+    Try { MockCromwellSwaggerClient.returnCopy(workflowDescriptions(wdl)) }
+  }
+
+}
+
+object MockCromwellSwaggerClient {
+  def returnCopy(wfDescription: WorkflowDescription): WorkflowDescription = {
```

Comment:
Added this because `workflowDescription` above was passing around a references to objects, which meant that tests were changing the WorkflowDescriptions in the TrieMap